server/src/main/scala/org/labrad/browser/NodeApi.scala

```diff
@@ -43,6 +46,12 @@ class NodeApi(cxn: LabradConnection)(implicit ec: ExecutionContext) extends Logg
     }
   }
 
+  def getNodeStatus(node: String): Future[NodeStatus] = {
+    cxn.to(node).call("status").map { servers =>
+      NodeStatus(node, getServerStatuses(servers))
+    }
```

Comment:
I'm trying to understand what happens if this call fails -- I see the `call` in `rpc.ts`, which returns a `Promise`. If this call fails here, and the returned `Future` is set to failure/error, I think I might be missing where the client side error ends up (i.e. I can't find anywhere where the Promise has a `.catch(...)` errback).